README.md

```diff
@@ -38,6 +38,8 @@ Visit [RBAC permissions for running the tasks](docs/tasks-rbac-permissions.md) i
 
 ## Usage
 
+#### copy-template, modify-vm-template and create-vm-from-template tasks are  deprecated and will be removed in future versions.
```

Comment:
please remove the extra space between `are  deprecated `

Reply:
Done